src/Command/RunCommand.php

```diff
@@ -17,15 +18,10 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * Class Command
- *
- * @package Povils\PHPMND\Console
- */
-class Command extends BaseCommand
+class RunCommand extends BaseCommand
 {
-    const EXIT_CODE_SUCCESS = 0;
-    const EXIT_CODE_FAILURE = 1;
+    public const SUCCESS = 0;
```

Comment:
renamed to be compatible with newest symfony/console https://github.com/symfony/console/blob/5.3/Command/Command.php#L34

we need our own constants as we supports symfony/console >=4.0,<5.1 versions and all of them don't have these constants.